lib/next_ls/extensions/elixir_extension.ex

```diff
@@ -111,4 +112,18 @@ defmodule NextLS.ElixirExtension do
   end
 
   def clamp(line), do: max(line, 0)
+
+  @unused_variable ~r/variable\s\"[^\"]+\"\sis\sunused/
```

Comment:
I wonder if elixir core would consider tagging diagnostics with IDs, like EX1 means "unused variable" 🤔

Reply:
That would be dope